tools/import_validate.py

```diff
@@ -34,19 +34,41 @@ def blank_import(data):
     if blanks[-1] == num - 1:
         blanks.pop(-1)
 
+def check_imports(data):
+    # We only tolerate blank lines between first and third party imports, as well as blank (_) imports
+    line = data.get('line', '')
+    import_state = data.get('import_state', TPIMPORTS)
+    # ignore . imports, but recognize 3rd party imports of the form github.com/user/proj
+    if not line.startswith('.') and '.' in line and import_state == FPIMPORTS:
```

Comment:
You need to trim whitespace before checking `not line.startswith('.')` or your script will fail for [dot imports](https://go.dev/ref/spec#Import_declarations).